pkg/controller/servicemesh/memberroll/controller.go

```diff
@@ -396,13 +400,25 @@ func (r *MemberRollReconciler) reconcileObject(ctx context.Context, roll *maistr
 }
 
 func isExcludedNamespace(ns string) bool {
+	// keep this in sync with getExcludedNamespaces()
 	return ns == common.GetOperatorNamespace() ||
 		ns == "kube" ||
 		ns == "openshift" ||
 		strings.HasPrefix(ns, "kube-") ||
 		strings.HasPrefix(ns, "openshift-")
 }
 
+func getExcludedNamespaces() []string {
+	// keep this in sync with isExcludedNamespace()
+	return []string{
+		common.GetOperatorNamespace(),
+		"kube",
```

Comment:
Kiali seems to use regex for this set: https://github.com/kiali/kiali-operator/blob/master/crd-docs/cr/kiali.io_v1alpha1_kiali.yaml#L22

Do we need to use `^` too?

Reply:
Good catch. Wonder why it seemed to work correctly when I manually tested this with Kiali. Anyway, I've now fixed this.

Reply:
It worked because it was a match. If we don't specify the `^` we are matching any part of the string.

My other question was if this is necessary or if Kiali takes care of this when namespaces is set to `**`.

Reply:
I mean, it correctly ignored all `openshift-xyz` namespaces, even though the regex was wrong (`"openshift-*"` instead of `"openshift-.*"`).

Kiali doesn't take care of this. You have to explicitly set excluded namespaces when you use `**`.